### ISSUE TITLE
Publishing in Steam-Workshop

### ISSUE THREAD

**Issue by hofq** [TTT-2/ttt2-acrylic-hud#2]
I would be happy if you would continue to develop the HUD and publish it into the Steam Workshop

**Comment by TimGoll** [comment]
I plan to finish this at some point, but I don't want to promise anything for now. I'm quite busy with the TTT2 coregame atm

**Comment by hofq** [comment]
Okay, I'll understand that. I am really like the HUD and it would be great on our private server. Thank you for the feedback

**Comment by hofq** [comment]
Do you think i can ever use it xD
(If you not planning to Finish it, maybe i fork it and try it myself :) )

**Comment by TimGoll** [comment]
Feel free to work on this and contribute to the project. Right now im focused on the core TTT2 gamemode

**Comment by hofq** [comment]
yeah, i looked a bid into it, but I'm just starting with programming in general and that's a bit too complex for the beginning
